src/backends/cgpm_backend.py

```diff
@@ -922,6 +923,76 @@ def logpdf_joint(
             multiprocess=self._multiprocess,
         )
 
+    # Return python data object that's ready to be written in JSON syntax
+    def json_ready_models(self, bdb, population_name):
```

Comment:
The methods of this class are designed to take in a `generator_id` not `population_name`.  Thus, please delete lines 928--932, which also removes the unnecessary assertion that only populations with a single generator can be dumped as JSON (those checks can be made at the site of the caller).

Reply:
I can't simply delete those 5 lines; the population_id and generator_id need to come from somewhere.  Are you saying the code should be moved to the caller(s)?